googlemediaframework/src/main/java/com/google/android/libraries/mediaframework/layeredvideo/PlaybackControlLayer.java

```diff
@@ -394,6 +399,7 @@ public PlaybackControlLayer(String videoTitle, FullscreenCallback fullscreenCall
     this.canSeek = true;
     this.fullscreenCallback = fullscreenCallback;
     this.shouldBePlaying = false;
+    this.savedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT;
```

Comment:
Does this assume the app starts with the phone in portrait mode? Should we grab the current orientation here instead of assuming portrait?